### PR TITLE
Update test_vis_clean.py after reorganization of hera_sim

### PR DESCRIPTION
**Pull request by jsdillon**
hera_sm change the interface to beam area interpolation and to `noise. sky_noise_jy`. This PR fixes a test to use the new interface. 

@nkern I'm also tagging you because I had to turn down the atol on one test to get it to work properly... for reasons I don't understand `assert not np.isclose(np.mean(np.abs(d1)[select]), np.mean(np.abs(d2)[select]), atol=10)` wasn't working. Is that unexepected?

### CODE REVIEW

**Review by r-pascua — APPROVED** [review]
The `hera_sim`-related changes look fine to me. I take it the commented out line (749 in `test_vis_clean.py`) is just there as a reference?

File: hera_cal/tests/test_vis_clean.py [hunk 1]
```diff
@@ -742,8 +744,10 @@ def test_trim_model(self):
         # delay width = intended delay width x (manually set dnu / original dnu of the attached data)
         np.random.seed(0)
         k = (23, 24, 'ee')
-        Op = noise.bm_poly_to_omega_p(V.freqs / 1e9)
-        V.data[k] += noise.sky_noise_jy(V.data[(23, 23, 'ee')], V.freqs / 1e9, V.lsts, Op, inttime=50)
+        beam_interp = Beam(HS_DATA_PATH / 'HERA_H1C_BEAM_POLY.npy')
+        Op = beam_interp(V.freqs / 1e9)
+        # V.data[k] += noise.sky_noise_jy(autovis=V.data[(23, 23, 'ee')], freqs=V.freqs / 1e9, lsts=V.lsts, omega_p=Op)
```
Comment (nkern) [inline hera_cal/tests/test_vis_clean.py:749]: can you rm this comment, unless its pertinent

**Comment by nkern** [comment]
I don't see why turning *down* the atol would make the test pass?

**Comment by nkern** [comment]
@jsdillon yes I'm fine w/ your change to the unit-test--that unit-test needs to be overhauled anyways b/c its not really testing what it should be

**Pull request merged**